app/controllers/concerns/first_step_form_concerns.rb

```diff
@@ -0,0 +1,20 @@
+# Methods shared by controllers which can serve Step 1
+# HiringStaff::Vacancies::JobSpecificationController is school-level users' Step 1
+module FirstStepFormConcerns
+  def set_up_form(form_class)
+    @form = form_class.new(form_params)
+  end
+
+  def save_form_params_on_vacancy_without_validation
+    @form.vacancy.status = :draft
+    Auditor::Audit.new(@form.vacancy, 'vacancy.create', current_session_id).log do
```

Comment:
I think it's worth talking with @stevenleggdfe about whether we can remove these sorts of things from the codebase, and rely on GA